### PR TITLE
[Security Center] Adding version to SecurityContact

### PR DESCRIPTION
**Pull request by chlahav**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by lirenhe** [comment]
@chlahav, could you take a look at all the check errors?

File: specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Create security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Create",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```
Comment (lirenhe) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json]: As only ApiVersion and subscriptionId are in the parameter list, what is the payload of this API?

File: specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+        "operationId": "SecurityContacts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json]: no need here

File: specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+      },
+      "required": [
+        "email",
+        "alertNotifications",
+        "alertsToAdmins"
+      ]
+    }
+  },
+  "parameters": {
+    "SecurityContactName": {
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json]: delete this parameter - it is not in use

File: specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json [hunk 4]
```diff
@@ -120,29 +111,20 @@
...
             "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
           },
           {
             "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
-          },
-          {
-            "$ref": "#/parameters/SecurityContactName"
-          },
-          {
-            "$ref": "#/parameters/SecurityContact"
```
Comment (KrisBash) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json]: This API appears to be missing the request body.

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comments

**Pull request closed without merging**